www/app/login/login.controller.js

```diff
@@ -0,0 +1,51 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('budgetrentacar.login')
+    .controller('LoginController', ['$scope', '$state', '$ionicPopup', '$localStorage', 'AuthService', function($scope, $state, $ionicPopup, $localStorage, AuthService) {
+      var ctrlScope = $scope;
+      ctrlScope.user = { };
+
+      ctrlScope.createAccount = function(){
+        $ionicPopup.alert({
+          tittle: 'Budget Rent a Car',
+          template: 'Not available in this prototype'  
+        }).then(function(res){
+          console.log('Show alert')
+        });
+      };
+
+      ctrlScope.login = function(){
+        console.log("login submit : " + ctrlScope.user);
+        AuthService.authWithPassword(ctrlScope.user)
+          .then(ctrlScope.authSuccess)
```

Comment:
mmm, authSuccess method has a non optional parameter, I recommend use explicit call for more readable code